src/main/java/uk/gov/ons/validation/service/InvokeConfig.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.ons.validation.service;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import uk.gov.ons.validation.entity.QuestionInputData;
+
+import static java.lang.String.format;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Log4j2
+public class InvokeConfig {
+
+    private String finalQuestCode;
+    private String finalDerivedQuestCode;
+    private String finalQuestCodeValue;
+    private String finalDerivedQuestValue;
+    private boolean isQuestionCodeFound;
+    private boolean isDerivedQuestFound;
+
+    private static final String MATCH_FOUND_QUEST = "Matching object found for Question and its code is %s.";
+    private static final String MATCH_FOUND_DERIVED_QUEST = "Matching object found for Derived Question and its code is %s.";
+
+    /**
+     * @param inputData
+     * @param configQuestionCode
+     */
+    public void processQuestionCodeFound(QuestionInputData inputData, String configQuestionCode) {
```

Comment:
I would have thought this method should just be called processQuestionCode.

Reply:
Done